test/DynamoCoreWpfTests/PresetViewModelTest.cs

```diff
@@ -58,5 +58,37 @@ public void TogglePresetOptionVisibility()
             //Check the Preset option visibility.
             Assert.AreEqual(false, ViewModel.EnablePresetOptions);
         }
+
+        [Test]
+        public void CanCreatePreset()
```

Comment:
sorry if I'm mistaken here, but where does this test check for preset creation? maybe the diff is incorrect on github, seems to be checking for the GetInputNodes() method